Header header
int8 status
float32 latitude
float32 longitude
float32 altitude
float32 HvitesseModule
float32 HvitesseAngle
float32 Vvitesse
uint8 satUsed
float64 timestamp